custom_model_runner/datarobot_drum/drum/language_predictors/r_predictor/score.R

```diff
@@ -96,6 +105,24 @@ load_serialized_model <- function(model_dir) {
     model
 }
 
+.load_data <- function(binary_data, mimetype=NULL, use_hook=TRUE) {
+    if (use_hook && !isFALSE(read_input_data_hook)) {
+        data <- read_input_data_hook(binary_data)
+    } else if (!is.null(mimetype) && mimetype == "text/mtx") {
```

Comment:
mimetype is null for normal CSV data or when?

Reply:
it's probably `text/csv`. I copied this code so I don't know for sure

Reply:
can you check and comment?

Reply:
it's null if it's dense

Reply:
so i'm deeming this code valid